### PR TITLE
First pass at some basic integration tests

### PR DESCRIPTION
**Pull request by Soliah**
Added first integration test that actually executes the project and launches a tmux session which gets asserted against.

Approach is what I suggested in #497. We would still need more coverage than this simple project though.

### CODE REVIEW

**Comment by adamstrickland** [comment]
Updated branch.  I agree that this falls short of what we *want* but maybe we merge it in to at least get the ball rolling?

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -22,6 +22,9 @@
 
 RSpec.configure do |config|
   config.order = "random"
+
+  # Don't run integration tests by default
+  # config.filter_run_excluding integration: true
```
Comment (ethagnawl) [inline spec/spec_helper.rb]: Should this line be commented out?

**Comment by ethagnawl** [comment]
Looks like a _good enough_ start to me!

**Comment by Soliah** [comment]
Hopefully the direction is clear - the implementation isn’t particularly good and I haven’t had time to flesh it out more.

File: spec/spec_helper.rb (hunk 1 above)
Comment (Soliah) [reply spec/spec_helper.rb]: I think these tests should be run separately maybe so yes.

**Comment by Soliah** [comment]
Also from memory the most difficult part was determining the state of the tmux session to assert against - especially across versions. capture_pane and stuff are only available in certain versions.

**Comment by adamstrickland** [comment]
Looks like this still needs some work.  Prolly should hold off for now.

**Comment by ethagnawl** [comment]
Should we close this for now?

**Pull request closed without merging**